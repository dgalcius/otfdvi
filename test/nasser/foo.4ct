\expandafter\ifx\csname doTocEntry\endcsname\relax \expandafter\endinput\fi 
\doTocEntry\toclikechapter{}{\csname a:TocLink\endcsname{1}{x1-1000}{QQ2-1-1}{Contents}}{3}\relax 
\doTocEntry\tocchapter{1}{\csname a:TocLink\endcsname{1}{x1-20001}{QQ2-1-2}{Introduction}}{5}\relax 
\doTocEntry\tocsection{1.1}{\csname a:TocLink\endcsname{1}{x1-30001.1}{QQ2-1-3}{some math}}{5}\relax 
\doTocEntry\tocthm{}{\csname a:TocLink\endcsname{1}{x1-3002}{}{{Theorem}{1}{Residue Theorem}}}{5}\relax 
\doTocEntry\tocsubsection{1.1.1}{\csname a:TocLink\endcsname{1}{x1-40001.1.1}{QQ2-1-4}{this is a subsection}}{6}\relax 
\doTocEntry\tocsubsubsection{}{\csname a:TocLink\endcsname{1}{x1-50001.1.1}{QQ2-1-5}{This is subsubsection with images}}{6}\relax 
\doTocEntry\toclof{1.1}{\csname a:TocLink\endcsname{1}{x1-5001r1}{}{\ignorespaces Contour $J(u)$}}{figure}\relax 
\doTocEntry\toclof{1.2}{\csname a:TocLink\endcsname{1}{x1-5002r2}{}{\ignorespaces Zooming on $J(u)$}}{figure}\relax 
\doTocEntry\tocchapter{2}{\csname a:TocLink\endcsname{1}{x1-60002}{QQ2-1-8}{This is a new chapter}}{11}\relax 
\doTocEntry\tocsection{2.1}{\csname a:TocLink\endcsname{1}{x1-70002.1}{QQ2-1-9}{This is a section for more math}}{11}\relax 
\doTocEntry\tocsubsection{2.1.1}{\csname a:TocLink\endcsname{1}{x1-80002.1.1}{QQ2-1-10}{problem 1}}{11}\relax 
\doTocEntry\tocsubsection{2.1.2}{\csname a:TocLink\endcsname{1}{x1-90002.1.2}{QQ2-1-11}{Example on page 500, textbook (Edwards\&Penny, 3rd edition)}}{12}\relax 
\par 
