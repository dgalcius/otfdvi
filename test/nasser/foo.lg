htfcss: cmbx  font-weight: bold;
htfcss: cmbxsl  font-weight: bold; font-style: oblique;
htfcss: cmmi font-style: italic;
htfcss: cmmib font-style: italic; font-weight: bold;
File: foo.html
File: foo.css
File: foo.tmp
File: foo.tmp
File: foo.tmp
File: foo.tmp
Css: p.noindent { text-indent: 0em }
Css: td p.noindent { text-indent: 0em; margin-top:0em; }
Css: p.nopar { text-indent: 0em; }
Css: p.indent{ text-indent: 1.5em }
Css: @media print {div.crosslinks {visibility:hidden;}}
Css: a img { border-top: 0; border-left: 0; border-right: 0; }
Font_Css("4"): .small-caps{font-variant: small-caps; }
Font_Css("10"): .htf-cmbx {font-weight: bold; font-style:normal;}
Font_Css("12"): .htf-calligraphy {font-family:cursive}
Font_Css("14"): .htf-italic {font-style: italic;}
Font_Css("16"): .htf-bold {font-weight: bold;}
Font_Css("12"): .htf-calligraphy-bold {font-family:cursive ; font-weight: bold; }
Css: center { margin-top:1em; margin-bottom:1em; }
Css: td center { margin-top:0em; margin-bottom:0em; }
Css: .Canvas { position:relative; }
Css: img.math{vertical-align:middle;}
Css: li p.indent { text-indent: 0em }
Css: li p:first-child{ margin-top:0em; }
Css: li p:last-child, li div:last-child { margin-bottom:0.5em; }
Css: li p~ul:last-child, li p~ol:last-child{ margin-bottom:0.5em; }
Css: .enumerate1 {list-style-type:decimal;}
Css: .enumerate2 {list-style-type:lower-alpha;}
Css: .enumerate3 {list-style-type:lower-roman;}
Css: .enumerate4 {list-style-type:upper-alpha;}
Css: div.newtheorem { margin-bottom: 2em; margin-top: 2em;}
Css: .obeylines-h,.obeylines-v {white-space: nowrap; }
Css: div.obeylines-v p { margin-top:0; margin-bottom:0; }
Css: .overline{ text-decoration:overline; }
Css: .overline img{ border-top: 1px solid black; }
Css: td.displaylines {text-align:center; white-space:nowrap;}
Css: .centerline {text-align:center;}
Css: .rightline {text-align:right;}
Css: div.verbatim {font-family: monospace; white-space: nowrap; text-align:left; clear:both; }
Css: .fbox {padding-left:3.0pt; padding-right:3.0pt; text-indent:0pt; border:solid black 0.4pt; }
Css: div.fbox {display:table}
Css: div.center div.fbox {text-align:center; clear:both; padding-left:3.0pt; padding-right:3.0pt; text-indent:0pt; border:solid black 0.4pt; }
Css: div.minipage{width:100%;}
Css: div.center, div.center div.center {text-align: center; margin-left:1em; margin-right:1em;}
Css: div.center div {text-align: left;}
Css: div.flushright, div.flushright div.flushright {text-align: right;}
Css: div.flushright div {text-align: left;}
Css: div.flushleft {text-align: left;}
Css: .underline{ text-decoration:underline; }
Css: .underline img{ border-bottom: 1px solid black; margin-bottom:1pt; }
Css: .framebox-c, .framebox-l, .framebox-r { padding-left:3.0pt; padding-right:3.0pt; text-indent:0pt; border:solid black 0.4pt; }
Css: .framebox-c {text-align:center;}
Css: .framebox-l {text-align:left;}
Css: .framebox-r {text-align:right;}
Css: span.thank-mark{ vertical-align: super }
Css: span.footnote-mark sup.textsuperscript, span.footnote-mark a sup.textsuperscript{ font-size:80%; }
Css: div.tabular, div.center div.tabular {text-align: center; margin-top:0.5em; margin-bottom:0.5em; }
Css: table.tabular td p{margin-top:0em;}
Css: table.tabular {margin-left: auto; margin-right: auto;}
Css: td p:first-child{ margin-top:0em; }
Css: td p:last-child{ margin-bottom:0em; }
Css: div.td00{ margin-left:0pt; margin-right:0pt; }
Css: div.td01{ margin-left:0pt; margin-right:5pt; }
Css: div.td10{ margin-left:5pt; margin-right:0pt; }
Css: div.td11{ margin-left:5pt; margin-right:5pt; }
Css: table[rules] {border-left:solid black 0.4pt; border-right:solid black 0.4pt; }
Css: td.td00{ padding-left:0pt; padding-right:0pt; }
Css: td.td01{ padding-left:0pt; padding-right:5pt; }
Css: td.td10{ padding-left:5pt; padding-right:0pt; }
Css: td.td11{ padding-left:5pt; padding-right:5pt; }
Css: table[rules] {border-left:solid black 0.4pt; border-right:solid black 0.4pt; }
Css: .hline hr, .cline hr{ height : 1px; margin:0px; }
Css: .tabbing-right {text-align:right;}
Css: span.TEX {letter-spacing: -0.125em; }
Css: span.TEX span.E{ position:relative;top:0.5ex;left:-0.0417em;}
Css: a span.TEX span.E {text-decoration: none; }
Css: span.LATEX span.A{ position:relative; top:-0.5ex; left:-0.4em; font-size:85%;}
Css: span.LATEX span.TEX{ position:relative; left: -0.4em; }
Css: div.float, div.figure {margin-left: auto; margin-right: auto;}
Css: div.float img {text-align:center;}
Css: div.figure img {text-align:center;}
Css: .marginpar {width:20%; float:right; text-align:left; margin-left:auto; margin-top:0.5em; font-size:85%; text-decoration:underline;}
Css: .marginpar p{margin-top:0.4em; margin-bottom:0.4em;}
Css: table.equation {width:100%;}
Css: .equation td{text-align:center; }
Css: td.equation { margin-top:1em; margin-bottom:1em; } 
Css: td.equation-label { width:5%; text-align:center; }
Css: td.eqnarray4 { width:5%; white-space: normal; }
Css: td.eqnarray2 { width:5%; }
Css: table.eqnarray-star, table.eqnarray {width:100%;}
Css: div.eqnarray{text-align:center;}
Css: div.array {text-align:center;}
Css: div.pmatrix {text-align:center;}
Css: table.pmatrix {width:100%;}
Css: span.pmatrix img{vertical-align:middle;}
Css: div.pmatrix {text-align:center;}
Css: table.pmatrix {width:100%;}
Css: span.bar-css {text-decoration:overline;}
Css: img.cdots{vertical-align:middle;}
Css: .partToc a, .partToc, .likepartToc a, .likepartToc {line-height: 200%; font-weight:bold; font-size:110%;}
Css: .chapterToc a, .chapterToc, .likechapterToc a, .likechapterToc, .appendixToc a, .appendixToc {line-height: 200%; font-weight:bold;}
Css: .index-item, .index-subitem, .index-subsubitem {display:block}
Css: div.caption {text-indent:-2em; margin-left:3em; margin-right:1em; text-align:left;}
Css: div.caption span.id{font-weight: bold; white-space: nowrap; }
Css: h1.partHead{text-align: center}
Css: p.bibitem { text-indent: -2em; margin-left: 2em; margin-top:0.6em; margin-bottom:0.6em; }
Css: p.bibitem-p { text-indent: 0em; margin-left: 2em; margin-top:0.6em; margin-bottom:0.6em; }
Css: .paragraphHead, .likeparagraphHead { margin-top:2em; font-weight: bold;}
Css: .subparagraphHead, .likesubparagraphHead { font-weight: bold;}
Css: .quote {margin-bottom:0.25em; margin-top:0.25em; margin-left:1em; margin-right:1em; text-align:justify;}
Css: .verse{white-space:nowrap; margin-left:2em}
Css: div.maketitle {text-align:center;}
Css: h2.titleHead{text-align:center;}
Css: div.maketitle{ margin-bottom: 2em; }
Css: div.author, div.date {text-align:center;}
Css: div.thanks{text-align:left; margin-left:10%; font-size:85%; font-style:italic; }
Css: div.author{white-space: nowrap;}
Font_Size: 11
Css: .quotation {margin-bottom:0.25em; margin-top:0.25em; margin-left:1em; }
Css: h1.partHead{text-align: center}
Css:  .chapterToc, .likechapterToc {margin-left:0em;}
Css:  .chapterToc ~ .likesectionToc, .chapterToc ~ .sectionToc, .likechapterToc ~ .likesectionToc, .likechapterToc ~ .sectionToc {margin-left:2em;}
Css:  .chapterToc ~ .likesectionToc ~ .likesubsectionToc, .chapterToc ~ .likesectionToc ~ .subsectionToc, .chapterToc ~ .sectionToc ~ .likesubsectionToc, .chapterToc ~ .sectionToc ~ .subsectionToc, .likechapterToc ~ .likesectionToc ~ .likesubsectionToc, .likechapterToc ~ .likesectionToc ~ .subsectionToc, .likechapterToc ~ .sectionToc ~ .likesubsectionToc, .likechapterToc ~ .sectionToc ~ .subsectionToc {margin-left:4em;}
Css: .chapterToc ~ .likesectionToc ~ .likesubsectionToc ~ .likesubsubsectionToc, .chapterToc ~ .likesectionToc ~ .likesubsectionToc ~ .subsubsectionToc, .chapterToc ~ .likesectionToc ~ .subsectionToc ~ .likesubsubsectionToc, .chapterToc ~ .likesectionToc ~ .subsectionToc ~ .subsubsectionToc, .chapterToc ~ .sectionToc ~ .likesubsectionToc ~ .likesubsubsectionToc, .chapterToc ~ .sectionToc ~ .likesubsectionToc ~ .subsubsectionToc, .chapterToc ~ .sectionToc ~ .subsectionToc ~ .likesubsubsectionToc, .chapterToc ~ .sectionToc ~ .subsectionToc ~ .subsubsectionToc, .likechapterToc ~ .likesectionToc ~ .likesubsectionToc ~ .likesubsubsectionToc, .likechapterToc ~ .likesectionToc ~ .likesubsectionToc ~ .subsubsectionToc, .likechapterToc ~ .likesectionToc ~ .subsectionToc ~ .likesubsubsectionToc, .likechapterToc ~ .likesectionToc ~ .subsectionToc ~ .subsubsectionToc, .likechapterToc ~ .sectionToc ~ .likesubsectionToc ~ .likesubsubsectionToc, .likechapterToc ~ .sectionToc ~ .likesubsectionToc ~ .subsubsectionToc, .likechapterToc ~ .sectionToc ~ .subsectionToc ~ .likesubsubsectionToc .likechapterToc ~ .sectionToc ~ .subsectionToc ~ .subsubsectionToc {margin-left:6em;}
Css:  .likesectionToc , .sectionToc {margin-left:0em;}
Css:  .likesectionToc ~ .likesubsectionToc, .likesectionToc ~ .subsectionToc, .sectionToc ~ .likesubsectionToc, .sectionToc ~ .subsectionToc {margin-left:2em;}
Css: .likesectionToc ~ .likesubsectionToc ~ .likesubsubsectionToc, .likesectionToc ~ .likesubsectionToc ~ .subsubsectionToc, .likesectionToc ~ .subsectionToc ~ .likesubsubsectionToc, .likesectionToc ~ .subsectionToc ~ .subsubsectionToc, .sectionToc ~ .likesubsectionToc ~ .likesubsubsectionToc, .sectionToc ~ .likesubsectionToc ~ .subsubsectionToc, .sectionToc ~ .subsectionToc ~ .likesubsubsectionToc, .sectionToc ~ .subsectionToc ~ .subsubsectionToc {margin-left:4em;}
Css:  .likesubsectionToc, .subsectionToc {margin-left:0em;}
Css:  .likesubsectionToc ~ .subsubsectionToc, .subsectionToc ~ .subsubsectionToc {margin-left:2em;}
Css: .equation td{text-align:center; }
Css: .equation-star td{text-align:center; }
Css: table.equation-star { width:100%; }
Css: table.equation { width:100%; }
Css: table.align, table.alignat, table.xalignat, table.xxalignat, table.flalign {width:95%; margin-left:5%; white-space: nowrap;}
Css: table.align-star, table.alignat-star, table.xalignat-star, table.flalign-star {margin-left:auto; margin-right:auto; white-space: nowrap;}
Css: td.align-label { width:5%; text-align:center; }
Css: td.align-odd { text-align:right; padding-right:0.3em;}
Css: td.align-even { text-align:left; padding-right:0.6em;}
Css: table.multline, table.multline-star {width:100%;}
Css: td.gather {text-align:center; }
Css: table.gather {width:100%;}
Css: div.gather-star {text-align:center;}
Css: .figure img.graphics {margin-left:10%;}
Css: .lstlisting .label{margin-right:0.5em; }
Css: div.lstlisting{font-family: monospace; white-space: nowrap; margin-top:0.5em; margin-bottom:0.5em; }
Css: div.lstinputlisting{ font-family: monospace; white-space: nowrap; }
Css: .lstinputlisting .label{margin-right:0.5em;}
Css: P.fancyvrb {white-space: nowrap; margin:0em;}
--- needs --- foo.idv[2] ==> foo0x.svg ---
--- needs --- foo.idv[3] ==> foo1x.svg ---
--- needs --- foo.idv[4] ==> foo2x.svg ---
--- needs --- foo.idv[5] ==> foo3x.svg ---
--- needs --- foo.idv[6] ==> foo4x.svg ---
--- needs --- foo.idv[7] ==> foo5x.svg ---
--- needs --- foo.idv[8] ==> foo6x.svg ---
--- needs --- foo.idv[9] ==> foo7x.svg ---
--- needs --- foo.idv[10] ==> foo8x.svg ---
--- needs --- foo.idv[12] ==> foo9x.svg ---
--- needs --- foo.idv[14] ==> foo10x.svg ---
--- needs --- foo.idv[16] ==> foo11x.svg ---
--- needs --- foo.idv[18] ==> foo12x.svg ---
--- needs --- foo.idv[20] ==> foo13x.svg ---
--- needs --- foo.idv[22] ==> foo14x.svg ---
--- needs --- foo.idv[23] ==> foo15x.svg ---
--- needs --- foo.idv[24] ==> foo16x.svg ---
--- needs --- foo.idv[25] ==> foo17x.svg ---
--- needs --- foo.idv[27] ==> foo18x.svg ---
--- needs --- foo.idv[28] ==> foo19x.svg ---
--- needs --- foo.idv[29] ==> foo20x.svg ---
--- needs --- foo.idv[30] ==> foo21x.svg ---
--- needs --- foo.idv[31] ==> foo22x.svg ---
--- needs --- foo.idv[32] ==> foo23x.svg ---
--- needs --- foo.idv[34] ==> foo24x.svg ---
--- needs --- foo.idv[35] ==> foo25x.svg ---
--- needs --- foo.idv[36] ==> foo26x.svg ---
--- needs --- foo.idv[37] ==> foo27x.svg ---
--- needs --- foo.idv[38] ==> foo28x.svg ---
--- needs --- foo.idv[39] ==> foo29x.svg ---
--- needs --- foo.idv[40] ==> foo30x.svg ---
--- needs --- foo.idv[41] ==> foo31x.svg ---
--- needs --- foo.idv[42] ==> foo32x.svg ---
--- needs --- foo.idv[44] ==> foo33x.svg ---
--- needs --- foo.idv[45] ==> foo34x.svg ---
--- needs --- foo.idv[46] ==> foo35x.svg ---
--- needs --- foo.idv[47] ==> foo36x.svg ---
--- needs --- foo.idv[48] ==> foo37x.svg ---
--- needs --- foo.idv[49] ==> foo38x.svg ---
--- needs --- foo.idv[50] ==> foo39x.svg ---
--- needs --- foo.idv[51] ==> foo40x.svg ---
--- needs --- foo.idv[52] ==> foo41x.svg ---
--- needs --- foo.idv[53] ==> foo42x.svg ---
--- needs --- foo.idv[54] ==> foo43x.svg ---
--- needs --- foo.idv[56] ==> foo44x.svg ---
--- needs --- foo.idv[57] ==> foo45x.svg ---
--- needs --- foo.idv[58] ==> foo46x.svg ---
--- needs --- foo.idv[59] ==> foo47x.svg ---
--- needs --- foo.idv[60] ==> foo48x.svg ---
--- needs --- foo.idv[61] ==> foo49x.svg ---
--- needs --- foo.idv[62] ==> foo50x.svg ---
--- needs --- foo.idv[63] ==> foo51x.svg ---
--- needs --- foo.idv[64] ==> foo52x.svg ---
--- needs --- foo.idv[66] ==> foo53x.svg ---
--- needs --- foo.idv[67] ==> foo54x.svg ---
--- needs --- foo.idv[68] ==> foo55x.svg ---
--- needs --- foo.idv[69] ==> foo56x.svg ---
--- needs --- foo.idv[70] ==> foo57x.svg ---
--- needs --- foo.idv[71] ==> foo58x.svg ---
--- needs --- foo.idv[72] ==> foo59x.svg ---
--- needs --- foo.idv[73] ==> foo60x.svg ---
--- needs --- foo.idv[74] ==> foo61x.svg ---
--- needs --- foo.idv[76] ==> foo62x.svg ---
--- needs --- foo.idv[77] ==> foo63x.svg ---
--- needs --- foo.idv[78] ==> foo64x.svg ---
--- needs --- foo.idv[79] ==> foo65x.svg ---
--- needs --- foo.idv[80] ==> foo66x.svg ---
--- needs --- foo.idv[81] ==> foo67x.svg ---
--- needs --- foo.idv[82] ==> foo68x.svg ---
--- needs --- foo.idv[83] ==> foo69x.svg ---
--- needs --- foo.idv[84] ==> foo70x.svg ---
--- needs --- foo.idv[85] ==> foo71x.svg ---
--- needs --- foo.idv[86] ==> foo72x.svg ---
--- needs --- foo.idv[87] ==> foo73x.svg ---
--- needs --- foo.idv[88] ==> foo74x.svg ---
--- needs --- foo.idv[89] ==> foo75x.svg ---
--- needs --- foo.idv[90] ==> foo76x.svg ---
--- needs --- foo.idv[91] ==> foo77x.svg ---
--- needs --- foo.idv[92] ==> foo78x.svg ---
--- needs --- foo.idv[93] ==> foo79x.svg ---
--- needs --- foo.idv[94] ==> foo80x.svg ---
tex4ht.c error: 21
--- characters ---
Font("cmr","10","10","109")
Font("fver8t","","10","98")
Font("fveb8t","","10","98")
Font("fver8t","","10","155")
Font("fver8t","","10","108")
Font("cmr","8","8","100")
Font("cmr","6","6","100")
Font("cmmi","8","8","100")
Font("cmsy","8","8","100")
Font("cmsy","6","6","100")
Font("fver8t","","10","90")
Font("fver8t","","10","54")
Font("fvmr8t","","10","98")
Font("fvero8t","","10","98")
Font("cmmi","10","10","109")
Font("cmsy","10","10","109")
Font("cmex","10","10","109")
Font("fver8t","","10","81")
Font("fvero8t","","10","81")
Font("cmsy","9","9","100")
Font("fver8c","","10","98")
Font("fver8t","","10","72")
Font("cmbx","10","10","109")
