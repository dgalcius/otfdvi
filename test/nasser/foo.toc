\select@language {english}
\contentsline {chapter}{\numberline {1}Introduction}{5}{chapter.1}
\contentsline {section}{\numberline {1.1}some math}{5}{section.1.1}
\contentsline {subsection}{\numberline {1.1.1}this is a subsection}{6}{subsection.1.1.1}
\contentsline {subsubsection}{This is subsubsection with images}{6}{section*.2}
\contentsline {chapter}{\numberline {2}This is a new chapter}{7}{chapter.2}
\contentsline {section}{\numberline {2.1}This is a section for more math}{7}{section.2.1}
\contentsline {subsection}{\numberline {2.1.1}problem 1}{7}{subsection.2.1.1}
\contentsline {subsection}{\numberline {2.1.2}Example on page 500, textbook (Edwards\&Penny, 3rd edition)}{7}{subsection.2.1.2}
