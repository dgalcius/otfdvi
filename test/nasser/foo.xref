\:CrossWord{)F1F-}{foo.html}{1}%
\:CrossWord{TITLE+}{My big title}{1}%
\:CrossWord{)M1x0}{;}{1}%
\:CrossWord{)Qx1-1000}{1}{3}%
\:CrossWord{)QQQ2-1-2}{1}{3}%
\:CrossWord{)QQQ2-1-3}{1}{3}%
\:CrossWord{)QQQ2-1-4}{1}{3}%
\:CrossWord{)QQQ2-1-8}{1}{3}%
\:CrossWord{)QQQ2-1-9}{1}{3}%
\:CrossWord{)QQQ2-1-10}{1}{3}%
\:CrossWord{)QQQ2-1-11}{1}{3}%
\:CrossWord{)Qx1-20001}{1}{5}%
\:CrossWord{)Qx1-30001.1}{1}{5}%
\:CrossWord{)Qx1-3002}{1}{5}%
\:CrossWord{)Qx1-40001.1.1}{1}{6}%
\:CrossWord{)Qx1-4001r1}{1}{6}%
\:CrossWord{)Qx1-4002r2}{1}{6}%
\:CrossWord{)Qx1-4003r3}{1}{6}%
\:CrossWord{)Qx1-4004r4}{1}{6}%
\:CrossWord{)Qx1-4005r5}{1}{6}%
\:CrossWord{)Qx1-4006r6}{1}{6}%
\:CrossWord{)Qx1-4007r7}{1}{6}%
\:CrossWord{)Qx1-4008r1}{1}{6}%
\:CrossWord{)Qx1-4009r2}{1}{6}%
\:CrossWord{)Qx1-4010r3}{1}{6}%
\:CrossWord{)Qx1-4011r4}{1}{6}%
\:CrossWord{)Qx1-4012r5}{1}{6}%
\:CrossWord{)Qx1-4013r6}{1}{6}%
\:CrossWord{)Qx1-4014r7}{1}{6}%
\:CrossWord{)Qx1-4015r8}{1}{6}%
\:CrossWord{)Qx1-4016r9}{1}{6}%
\:CrossWord{)Qx1-50001.1.1}{1}{6}%
\:CrossWord{)Qx1-5001r1}{1}{7}%
\:CrossWord{)Qx1-5002r2}{1}{7}%
\:CrossWord{1cAp0}{x1-5001r1}{8}%
\:CrossWord{1cAp1}{x1-5002r2}{8}%
\:CrossWord{)Qx1-60002}{1}{11}%
\:CrossWord{)Qx1-6002r1}{1}{11}%
\:CrossWord{)Qx1-6004r2}{1}{11}%
\:CrossWord{)Qx1-6006r3}{1}{11}%
\:CrossWord{)Qx1-6008r4}{1}{11}%
\:CrossWord{)Qx1-6010r5}{1}{11}%
\:CrossWord{)Qx1-6012r6}{1}{11}%
\:CrossWord{)Qx1-70002.1}{1}{11}%
\:CrossWord{)Qx1-80002.1.1}{1}{11}%
\:CrossWord{)M2x10}{;}{11}%
\:CrossWord{)M3x10}{;}{12}%
\:CrossWord{)Qx1-90002.1.2}{1}{12}%
\:CrossWord{)M4x11}{;}{12}%
\:CrossWord{)M5x11}{;}{13}%
\:CrossWord{)M6x11}{;}{14}%
\:CrossWord{)M7x11}{;}{15}%
\:CrossWord{)M8x11}{;}{15}%
\:CrossWord{)M9x11}{;}{16}%
