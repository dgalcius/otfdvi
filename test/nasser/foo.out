\BOOKMARK [0][-]{chapter.1}{Introduction}{}% 1
\BOOKMARK [1][-]{section.1.1}{some math}{chapter.1}% 2
\BOOKMARK [2][-]{subsection.1.1.1}{this is a subsection}{section.1.1}% 3
\BOOKMARK [0][-]{chapter.2}{This is a new chapter}{}% 4
\BOOKMARK [1][-]{section.2.1}{This is a section for more math}{chapter.2}% 5
\BOOKMARK [2][-]{subsection.2.1.1}{problem 1}{section.2.1}% 6
\BOOKMARK [2][-]{subsection.2.1.2}{Example on page 500, textbook \(Edwards\046Penny, 3rd edition\)}{section.2.1}% 7
